packages/synapse-react-client/src/components/dataaccess/SubmissionPage/SubmissionPage.tsx

```diff
@@ -0,0 +1,542 @@
+}
+
+function DataAccessSubmissionFileHandleLink(props: {
+  submissionId: string
+  fileHandleId: string
+}) {
+  const { submissionId, fileHandleId } = props
+  const fileHandleAssociation = useMemo(
+    () => ({
+      fileHandleId: fileHandleId,
+      associateObjectId: submissionId,
+      associateObjectType:
+        FileHandleAssociateType.DataAccessSubmissionAttachment,
+    }),
+    [fileHandleId, submissionId],
+  )
+  return (
+    <FileHandleLink
+      key={fileHandleId}
+      showDownloadIcon={true}
+      fileHandleAssociation={fileHandleAssociation}
+    />
+  )
+}
+
+type AccessRequirementWikiType = {
+  accessRequirementId: string
+}
+
+function AccessRequirementWiki(props: AccessRequirementWikiType) {
+  const { data: wikiPageKey } = useGetAccessRequirementWikiPageKey(
+    props.accessRequirementId,
+    {
+      throwOnError: true,
+    },
+  )
+
+  return wikiPageKey ? (
+    <div className="AccessRequirementWikiContainer">
+      <div className="AccessRequirementWikiContent">
+        <Typography variant="headline1">Access Requirement</Typography>
+        <hr />
+        <MarkdownSynapse
+          wikiId={wikiPageKey?.wikiPageId}
+          ownerId={wikiPageKey?.ownerObjectId}
+          objectType={wikiPageKey?.ownerObjectType}
+        />
+      </div>
+    </div>
+  ) : (
+    <Skeleton width={'100%'} height={'600px'} />
+  )
+}
+
+/**
+ * returns true iff the userId is a requester and/or listed on the accessor changes on the submission.
+ * @param userId
+ * @param submission
+ */
+function isRequesterOnSubmission(userId: string, submission: Submission) {
+  return (
+    submission.submittedBy === userId ||
+    Boolean(submission.accessorChanges.find(acc => acc.userId === userId))
+  )
+}
+
+/**
+ * Page for a Data Access Submission. Supports the following user types:
+ *  - ACT member or other designated reviewer can view, and choose to approve or reject.
+ *  - Data Access Request submitter can view and modify their own submission.
+ *  - Data Access Request requester (who is not a submitter) can view their own submission.
+ */
+export default function SubmissionPage(props: SubmissionPageProps) {
+  const { submissionId, isReviewer } = props
+  const [showRejectionDialog, setShowRejectionDialog] = useState(false)
+
+  const { data: currentUserProfile } = useGetCurrentUserProfile()
+
+  const { data: submission, isLoading: isLoadingSubmission } =
+    useGetDataAccessSubmission(submissionId)
+  const isRequester = Boolean(
+    submission &&
+      currentUserProfile &&
+      isRequesterOnSubmission(currentUserProfile.ownerId, submission),
+  )
+
+  const { data: accessApproval, isLoading: isLoadingAccessApproval } =
+    useGetUserAccessApproval(String(submissionId), {
+      enabled:
+        isRequester &&
+        submission &&
+        submission.state === SubmissionState.APPROVED,
+    })
+
+  const isExpired = Boolean(
+    accessApproval && dayjs(accessApproval.expiredOn).isBefore(dayjs()),
+  )
```

Comment:
Requesters can now get the AccessApproval for the submission. We can use the AccessApproval to determine if it's expired